src/backend/src/prisma/migrations/20250127014440_finance_redesign/migration.sql

```diff
@@ -1,3 +1,140 @@
+/*
+  Warnings:
+  - You are about to drop the column `allowedRefundSources` on the `Account_Code` table. All the data in the column will be lost.
+  - You are about to drop the column `otherReason` on the `Reimbursement_Product_Reason` table. All the data in the column will be lost.
+  - You are about to drop the column `account` on the `Reimbursement_Request` table. All the data in the column will be lost.
+  - Added the required column `indexCodeId` to the `Reimbursement_Request` table without a default value. This is not possible if the table is not empty.
+  - Added the required column `passwordHash` to the `Vendor` table without a default value. This is not possible if the table is not empty.
+  - Added the required column `taxExempt` to the `Vendor` table without a default value. This is not possible if the table is not empty.
+  - Added the required column `username` to the `Vendor` table without a default value. This is not possible if the table is not empty.
+*/
+-- CreateTable
+CREATE TABLE "Index_Code" (
+    "indexCodeId" TEXT NOT NULL,
+    "name" TEXT NOT NULL,
+    "dateCreated" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "dateDeleted" TIMESTAMP(3),
+    "userCreatedId" TEXT NOT NULL,
+    "userDeletedId" TEXT,
+
+    CONSTRAINT "Index_Code_pkey" PRIMARY KEY ("indexCodeId")
+);
+
+-- CreateTable
+CREATE TABLE "Reimbursement_Product_Other_Reason" (
+    "otherReimbursementProductReasonId" TEXT NOT NULL,
+    "name" TEXT NOT NULL,
+    "dateCreated" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "dateDeleted" TIMESTAMP(3),
+    "userCreatedId" TEXT NOT NULL,
+    "userDeletedId" TEXT,
+    "budget" INTEGER NOT NULL,
+    "indexCodeId" TEXT NOT NULL,
+    "accountCodeId" TEXT NOT NULL,
+
+    CONSTRAINT "Reimbursement_Product_Other_Reason_pkey" PRIMARY KEY ("otherReimbursementProductReasonId")
+);
+-- AlterTable
+ALTER TABLE "Reimbursement_Request" ADD COLUMN     "indexCodeId" TEXT NOT NULL;
+
+UPDATE "Reimbursement_Request" AS rr
+SET "indexCodeId" = (
+    SELECT "indexCodeId"  
+    FROM "Index_Code" AS ic
+    WHERE ic."indexCodeId" = rr."indexCodeId"
+)
+WHERE rr."indexCodeId" IS NOT NULL;
+
+-- AlterTable
+ALTER TABLE "Reimbursement_Product_Reason" ADD COLUMN     "otherReasonId" TEXT;
+
+INSERT INTO "Index_Code" ("indexCodeId", "name", "dateCreated", "userCreatedId")
+SELECT gen_random_uuid(), "Account_Code"."name", NOW(), 'default-user-id'
+FROM "Reimbursement_Request"
```

Comment:
you dont need this from. you can just replace account_code.name with “Cash”, and then copy paste it and do it with budget